comms/src/protocol/protocols.rs

```diff
@@ -110,6 +113,7 @@ mod test {
     fn add() {
         let (tx, _) = mpsc::channel(1);
         let protos = [
+            IDENTITY_PROTOCOL.clone(),
```

Comment:
Because its small, would it not be better to implement copy for this?

Reply:
It's a `bytes::Bytes`, so I'd have to implement a new type which I don't think is worth it - this is inside the `add` function, so anyone calling this will not have to clone.